graphql-dgs-spring-webmvc-autoconfigure/src/main/kotlin/com/netflix/graphql/dgs/webmvc/autoconfigure/GraphiQLConfigurer.kt

```diff
@@ -49,13 +51,14 @@ open class GraphiQLConfigurer(
     }
 
     override fun addResourceHandlers(registry: ResourceHandlerRegistry) {
+        val graphqlPath = servletContext.contextPath + configProps.path
```

Comment:
Info logging that hints the developer where the endpoint is available?

Reply:
Done.